src/main/java/it/teamdigitale/ndc/harvester/model/ControlledVocabularyModel.java

```diff
@@ -6,37 +6,45 @@
 import it.teamdigitale.ndc.harvester.exception.InvalidAssetException;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
 import org.apache.jena.rdf.model.Property;
 import org.apache.jena.rdf.model.Resource;
 import org.apache.jena.rdf.model.Statement;
 import org.apache.jena.rdf.model.StmtIterator;
 import org.apache.jena.vocabulary.DCTerms;
+import org.apache.jena.vocabulary.RDF;
 
 @Slf4j
 public class ControlledVocabularyModel extends BaseSemanticAssetModel {
-    public static final String KEY_CONCEPT_IRI =
-        "https://w3id.org/italia/onto/ndc-profile/keyConcept";
+
+    private static final String NDC_API_URI = "https://w3id.org/italia/controlled-vocabulary-api/";
```

Comment:
@ash1425 wdyt to move all constants to a separate file, so that they are reusable?